newIDE/app/src/ExportAndShare/GenericExporters/OnlineWebExport/OnlineGameLink.js

```diff
@@ -103,25 +108,6 @@ const OnlineGameLink = ({
     [exportPending, timeBeforeExportFinished]
   );
 
-  const loadGame = React.useCallback(
```

Comment:
simplified this component so that it uses the game loaded on the PublishHome, rather than load it again

Reply:
Perfect